_test/ensure-lib-src-rs-exist.sh

```diff
@@ -4,16 +4,46 @@ repo=$(cd "$(dirname "$0")/.." && pwd)
 
 missing=""
 
+empty_stub=""
+
+check_status=0
+
+IGNORED_EXERCISES=(
+	"two-fer" #deprecated
+	"nucleotide-codons" #deprecated
+	"hexadecimal" #deprecated
+)
+
 for dir in $repo/exercises/*/; do
-  exercise=$(basename "$dir")
```

Comment:
are these changes purely for re-indenting the lines?

I would ask that re-indent be its own commit (or not included at all)

Reply:
May I ask why?
The lines I have added for this PR have their own indent and if i leave the above line as they were, it would hurt readability.

Reply:
> May I ask why?

Well, I thought that if a line is changed in a way that isn't consistent with the commit message, it would cause confusion, and make it harder to understand the git history.

> if i leave the above line as they were, it would hurt readability.

indeed, so I think I should not suggest they are left as they are. Putting the indentation change in its own commit should accomplish both the goals though (understand history, and keep the indentation consistent)

Reply:
Note that I understand I am causing non-trivial hardship here, by asking for the re-indenting be its own commit. It is not necessarily the most pleasant experience to have to redo the commits to split them in two.

I offer this remedy:

Since I'm the one who cares most about this, I offer to make it happen. However I can only execute this offer at a time more convenient to me. Luckily we still have time before all exercises have their stubs, so we are not in a huge hurry.

There is no obligation to accept this offer, of course.

Reply:
> make it harder to understand the git history

and make it harder to review. Now a reviewer needs to think to see whether a line was changed *only* for reindenting, or if it has a change other than reindent. A separate commit makes it clear to the reviewer.